cli/auth.go

```diff
@@ -108,13 +108,13 @@ func LoginAction(cCtx *cli.Context) error {
 
 func (c *viamClient) loginAction(cCtx *cli.Context) error {
 	loggedInMessage := func(t *token, alreadyLoggedIn bool) {
-		already := "already "
+		already := "Already l"
 		if !alreadyLoggedIn {
-			already = ""
+			already = "L"
 			viamLogo(cCtx.App.Writer)
 		}
 
-		fmt.Fprintf(cCtx.App.Writer, "%slogged in as %q, expires %s\n", already, t.User.Email,
+		printf(cCtx.App.Writer, "%sogged in as %q, expires %s", already, t.User.Email,
```

Comment:
:joy: 

Might just be better to duplicate the printf. Fine as it is though.

Reply:
👏 

[ignorable] fwiw another approach would be to capitalize the first letter inside the printf function (could control it with a boolean if you want)

Reply:
Thanks for suggestions! I think I'll just leave as-is for now; the auto-capitalizing logic is kind of confusing and will make it hard to grep for messages in the CLI. I don't even like having it for errors, but 🤷🏻 . I think I'd rather devs write messages here and have them printed as close to what they're writing as possible.